Apps/Common/test/unit/Services/Email_Test.cs

```diff
@@ -56,6 +56,7 @@ public void ShouldProcessTemplate()
                 new Mock<IMessagingVerificationDelegate>().Object,
                 new Mock<IWebHostEnvironment>().Object);
             Email actual = emailService.ProcessTemplate(emailTo, template, d);
+            expected.Id = actual.Id;
```

Comment:
is this done because we dont know a-priori the id?

Reply:
yep!